### PR TITLE
chore: use npx in pre-commit hook

### PR DESCRIPTION
**Pull request by SimenB**
#### Purpose (TL;DR) - mandatory

See https://github.com/sinonjs/fake-timers/pull/440/files#r1032069294

https://typicode.github.io/husky/#/?id=locally-installed-binaries

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-$(npm bin)/lint-staged
+npx --no lint-staged
```
Comment (SimenB) [inline .husky/pre-commit:4]: we could use `npm exec` as well, but `npx` should be installed on all supported versions of node

Comment (fatso83) [reply .husky/pre-commit:4]: What kind of NPM do you have installed that does not have the `bin` subcommand? What does `npm --version` say if `npm bin` fails?

Comment (SimenB) [reply .husky/pre-commit:4]: It was removed in npm v9: https://github.com/npm/cli/releases/tag/v9.0.0

Comment (fatso83) [reply .husky/pre-commit:4]: Ah, OK, that seals the deal for me.

**Review by fatso83 — APPROVED** [review]

**Pull request merged**